n1netails-api/pom.xml

```diff
@@ -112,7 +112,10 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-
+		<dependency>
```

Comment:
would you be able to place this dependency with the other spring dependencies maybe between line 57 and 58